src/App.js

```diff
@@ -11,25 +11,69 @@ import ListView from './pages/listview/ListView';
 import GraphView from './pages/graphview/GraphView';
 import Notebook from './pages/notepage/Notebook';
 import TeTuMenu from "./components/menu/Menu";
+import {useState} from 'react'
 
 function App() {
+   const [user, setUser] = useState(JSON.parse(localStorage.getItem('user') || null));
+
+   const ProtectedRoute = ({ user, redirectPath = "/", children }) => {
+     if (!user) {
+       return <Navigate to={redirectPath} replace />;
+     }
+
+     return children ? children : <Outlet />;
+   };
+
    return (
-      <PayPalScriptProvider options={{ "client-id": "AYoTMzxCm2u7ot7z8kCWQGMgUlf79LmiJIrLY2zBpIemVUhBZh3nhF4llvBr5bIAk4jNgey06AAae5ex" }}>
-         <BrowserRouter>
-            <Routes>
-               <Route path="/" element={<LandingPage />} />
-               <Route path="/home" element={<Home />} />
-               {/*<Route path="/note" element={<NotePage />} />*/}
-               <Route path="/note/:pageId" element={<Notebook />} />
-               <Route path="/test2" element={<TeTuMenu />} />
-               <Route path="/file" element={<FilePage />} />
-               <Route path="/test" element={<DraggableEditor />} />
-               <Route path="/pricing" element={<PricingPage />} />
-               <Route path="/table" element={<ListView />} />
-               <Route path="/graph" element={<GraphView />} />
-            </Routes >
-         </BrowserRouter >
-      </PayPalScriptProvider >
+     <PayPalScriptProvider
+       options={{
+         "client-id":
+           "AYoTMzxCm2u7ot7z8kCWQGMgUlf79LmiJIrLY2zBpIemVUhBZh3nhF4llvBr5bIAk4jNgey06AAae5ex",
+       }}
+     >
+       <BrowserRouter>
```

Comment:
create privateRoutes component then wrap the path

Reply:
I have issue please fix it